doc/users/plotting/colormaps/lightness.py

```diff
@@ -1,120 +1,100 @@
 '''
+==========================
+Lightness of the colormaps
+==========================
+
 For each colormap, plot the lightness parameter L* from CIELAB colorspace
 along the y axis vs index through the colormap. Colormaps are examined in
 categories as in the original matplotlib gallery of colormaps.
 '''
 
-from colormaps import cmaps
 import numpy as np
+import matplotlib as mpl
 import matplotlib.pyplot as plt
 from matplotlib import cm
-import matplotlib as mpl
 from colorspacious import cspace_converter
+from colormaps import cmaps  # the colormaps, grouped by category
 
 mpl.rcParams.update({'font.size': 12})
 
-# indices to step through colormap
+# Number of colormap per subplot for particular cmap categories
+_DSUBS = {'Perceptually Uniform Sequential': 4, 'Sequential': 6,
+          'Sequential (2)': 6, 'Diverging': 6, 'Qualitative': 4,
+          'Miscellaneous': 6}
+
+# Spacing between the colormaps of a subplot
```

Comment:
I think it makes more sense to change the figure size like it used to be done. Now you get a different sized Axes and different number of tweaks for each figure.